projects/weather-app.md

```diff
@@ -96,8 +97,8 @@ When you press the submit button on the form you will get data from the API. Mak
 
 ## Guide Code
 
-You can see a guide code working with hard-code data [here](https://github.com/Yosolita1978/HardCodeDataWeatherApi)
-You can see a guide code working with real data with a API_KEY [here](https://github.com/Yosolita1978/RealDataWeatherAPI)
+- You can see a guide code working with hard-code data [here](https://github.com/Yosolita1978/HardCodeDataWeatherApi).
+- You can see a guide code working with real data with a API_KEY [here](https://github.com/Yosolita1978/RealDataWeatherAPI).
```

Comment:
You can see an example of working with real data coming from the use of an API_KEY [here]

Reply:
Thanks for pointing this out! Just committed the suggested change, the commit # for that is 7f00590e60a03023c0b312567f065126b918a4a8.